config/prow/testgrid/testgrid.yaml

```diff
@@ -303,6 +303,28 @@ test_groups:
 - name: ci-knative-net-istio-test-coverage
   gcs_prefix: knative-prow/logs/ci-knative-net-istio-go-coverage
   short_text_metric: "coverage"
+- name: ci-knative-net-kourier-continuous
+  gcs_prefix: knative-prow/logs/ci-knative-net-kourier-continuous
+  alert_stale_results_hours: 3
+- name: ci-knative-net-kourier-nightly-release
+  gcs_prefix: knative-prow/logs/ci-knative-net-kourier-nightly-release
+  alert_options: 
```

Comment:
<!--whitespace-bot-->
Remove trailing whitespace:
```suggestion
  alert_options:
```

Reply:
those changes were added by `make config` I guess, this should be changed on the template?

Reply:
that's correct, feel free give it a shot

Reply:
templates are here: https://github.com/knative/test-infra/tree/master/tools/config-generator/templates

Reply:
This one cannot be fixed from the templates, it was due to the change in https://github.com/knative/test-infra/pull/1719 and I haven't figured out how to fix it.
Let's merge this PR first and I will take a further look.

Reply:
I've been able to remove any trailing whitespace by modfiying the outputConfig function in the config generator toool. 

```go
// outputConfig outputs the given line, if not empty, to stdout.
func outputConfig(line string) {
	if strings.TrimSpace(line) != "" {
		fmt.Fprintln(output, strings.TrimRight(line, " "))
		emittedOutput = true
	}
}
```

not the nicest way of doing it perhaps... but, should I open a PR with those changes?

Reply:
well, just opened a PR with that change, feel free to close it if it's not the proper way of doing it: 
https://github.com/knative/test-infra/pull/1826

Reply:
Let's wait for @chizhg to fix it, he is more familiar with it